### DISCUSSION TITLE
Unable to log into development version

### DISCUSSION THREAD

**Discussion by andersmmg** [outline/outline#3755] [Self Hosting]
I am unable to use any of the auth methods I tested to log in. All of them just bring me back to the login screen with no output, and the yarn output gives an error: 
`ClientError [UnauthorizedError]: The maximum number of teams has been reached`
I have no teams yet (as I can't even log in) and both the `teams` and `authentication_providers` tables are still empty.

**To Reproduce**
1. Download and set up development environment
2. Add an authentication provider
3. Try to log in, get nothing
4. See error in terminal output

**Expected behavior**
It should just log in as usual.

**Outline (please complete the following information):**
- Install: development environment
- Version: [32b7a7d](https://github.com/outline/outline/commit/32b7a7df00c267d9bf3e38e51da1104a1995238f)

**Desktop (please complete the following information):**
 - OS: Manjaro Linux
 - Browser Chrome
 - Version 103.0.5060.53

NOTE: I did try [this](https://github.com/outline/outline/issues/2542#issuecomment-917324993) which worked, but according the next comment this is not recommended.

**Comment by tommoor** [comment]
> The maximum number of teams has been reached

You can only receive this error if there is already a team in the database and you're trying to login with a different authentication method than the one it has. This confusion will be cleared up quite a lot in the next release.

**Reply by andersmmg** [reply]
Where would I see what the team is? As I said originally, the teams database table (and most of the tables) is completely empty

**Reply by tommoor** [reply]
I guess based on this logic if your authentication provider does not provide a domain somehow then it could result in the error – https://github.com/outline/outline/blob/v0.64.4/server/commands/teamCreator.ts#L63 – this would be a first though

What auth provider are you using

**Reply by andersmmg** [reply]
I've tried this with Slack and OIDC, same results

**Reply by andersmmg** [reply]
The linked code does make sense, since enabling that env variable fixes it

**Reply by tommoor** [reply]
Which env variable – If you mean `HOSTED` that is intended for use on the multi-tenant getoutline.com, your behavior will be strange in a multitude of other ways with it enabled

**Reply by andersmmg** [reply]
Yeah, that's why I don't want to have it enabled. I used it to test if if helped as I've seen others do and it did
